src/geopackage/CMakeLists.txt

```diff
@@ -0,0 +1,24 @@
+cmake_minimum_required(VERSION 3.10)
+
+
+string(COMPARE EQUAL "${CMAKE_CXX_COMPILER_ID}" "IntelLLVM" _cmp)
+if (NOT _cmp)
+    message(WARNING "[NGen::geopackage] geometry.cpp cannot be optimized with "
+                    "${CMAKE_CXX_COMPILER_ID} due to a suspected compiler/boost issue. "
```

Comment:
True... this will affect us in ops... on the other hand the de-optimization is limited to one file, and I doubt this will be a major time sink?

Also might be worth cross-checking with #559 if we upgrade our boost version.

Reply:
If we do end up needing to debug it, I've got more than my fair share of experience working out compiler bugs and relevant work-arounds.

Reply:
Also, do I understand correctly that this code is used in startup, and not while we're repeatedly time-stepping?

Reply:
@PhilMiller Yes, that is correct. It's used when reading in the features at the beginning, i.e. here: https://github.com/NOAA-OWP/ngen/blob/e8f46463d73e3652cde57d9f5346b516655f0c6a/src/NGen.cpp#L266-L287